docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala

```diff
@@ -72,9 +73,62 @@ object MyImplicitExceptionHandler {
 
     Http().bindAndHandle(route, "localhost", 8080)
   }
+
   //#implicit-handler-example
 }
 
+object RespondWithHeaderExceptionHandlerExample {
+  //#respond-with-header-exceptionhandler-example
+  import akka.actor.ActorSystem
+  import akka.http.scaladsl.model.HttpResponse
+  import akka.http.scaladsl.model.StatusCodes._
+  import akka.http.scaladsl.model.headers.RawHeader
+  import akka.http.scaladsl.server._
+  import Directives._
+  import akka.http.scaladsl.Http
+  import akka.stream.ActorMaterializer
+  import RespondWithHeaderExceptionHandler.route
+
+
+  object RespondWithHeaderExceptionHandler {
+    implicit def myExceptionHandler: ExceptionHandler =
+      ExceptionHandler {
+        case _: ArithmeticException =>
+          extractUri { uri =>
+            println(s"Request to $uri could not be handled normally")
+            complete(HttpResponse(InternalServerError, entity = "Bad numbers, bad result!!!"))
```

Comment:
This is me being super nitpicking, but can you write the same handling for both Java and Scala.
Scala version prints and completes, Java one just completes.
Also, maybe they should return the same text :)

Reply:
That is an important thing btw, definitely good that you caught it :)